### PR TITLE
Add LongAP One support

### PR DESCRIPTION
**Pull request by timcooijmans**

### CODE REVIEW

File: src/utils/stakingClient.ts
Comment (tyler-whitman) [inline src/utils/stakingClient.ts:68]: We will need to wait to confirm that maker ID 7 will be correct.

File: src/features/hotspots/setup/HotspotSelectionListItem.tsx
Comment (tyler-whitman) [inline src/features/hotspots/setup/HotspotSelectionListItem.tsx:15]: This should to be `longap-one.svg`

Comment (tyler-whitman) [reply src/features/hotspots/setup/HotspotSelectionListItem.tsx:15]: Also can you remove the extra space after from

**Comment by cokes518** [comment]
Thanks Tim. App team will do a quick review. Just FYI, we'll merge this to main when community has approved Long AP One as a Maker.

**Comment by cokes518** [comment]
Looks like the unscientific poll on Discord has approved Long AP. I'll wait for DeWi to make the official announcement before merging.

File: src/features/hotspots/setup/HotspotSelectionListItem.tsx
Comment (timcooijmans) [reply src/features/hotspots/setup/HotspotSelectionListItem.tsx]: Fixed

**Comment by timcooijmans** [comment]
@tyler-whitman how can we get this merged? Maker-ID 7 is confirmed to be ours.

File: src/utils/stakingClient.ts
Comment (cokes518) [reply src/utils/stakingClient.ts:68]: 7 is correct.

**Comment by cokes518** [comment]
@tyler-whitman we can merge this. thanks!!

**Review by tyler-whitman — APPROVED** [review]

**Comment by tyler-whitman** [comment]
Just waiting for one more review on this. @matthewcarlreetz

**Review by allenan — APPROVED** [review]

**Pull request merged**
